Dockerfile-linux.template

```
@@ -221,8 +241,20 @@ RUN set -eux; \
 		gpgconf --kill all; \
 		rm -rf "$GNUPGHOME"; \
 	fi; \
+{{ if .version == "8.4.0alpha1" then ( -}}
+	\
+	# Add patch; see https://github.com/docker-library/php/pull/1526
+	curl -fsSL -o php-pdo.patch https://patch-diff.githubusercontent.com/raw/php/php-src/pull/14797.patch; \
+	echo "d9f33beda1ffffc66299377a0946b1fe2916382e320084568ae6ee2a6d1fb19e php-pdo.patch" | sha256sum -c -; \
```

This isn't using the `ENV`s that you added, but I think that's probably fine since it makes the amount of things to remove later smaller (and I don't think the `ENV`s add much value to downstream users). :smile:

I do have a few notes here though:
- we should include the undocumented `full_index=1` to help combat GitHub non-reproducibility
- single quotes
- `*` in sha256sum to avoid text-mode parsing

```suggestion
	curl -fsSL -o php-pdo.patch 'https://github.com/php/php-src/pull/14797.patch?full_index=1'; \
	echo '3a95762048a56ec0f59cb9ee18df49751ffb0bdd0e91e021b57f69ac7af62996 *php-pdo.patch' | sha256sum -c -; \
```